app/actions/websocket/index.ts

```diff
@@ -137,6 +137,9 @@ async function doReconnect(serverUrl: string) {
     const currentChannel = await getCurrentChannel(database);
     const currentActiveServerUrl = await getActiveServerUrl();
 
+    if (serverUrl === currentActiveServerUrl) {
+        DeviceEventEmitter.emit(Events.FETCHING_POSTS, true);
+    }
```

Comment:
Is there anywhere else where this needs to be set as true ?

Reply:
I don't think so. The entry logic happens in:
- App entry (the channel list component has been just instantiated, so it is set as true)
- Login entry (ídem)
- Notification entry (ídem)
- Websocket reconnect (solved with this)

Unless there is any circumstance where app, login or notification entry can happen with the ChannelList component already instantiated, it should not be needed.